ament_lint_common/package.xml

```diff
@@ -16,6 +16,7 @@
   <exec_depend>ament_cmake_copyright</exec_depend>
   <exec_depend>ament_cmake_cppcheck</exec_depend>
   <exec_depend>ament_cmake_cpplint</exec_depend>
+  <exec_depend>ament_cmake_flake8</exec_depend>
```

Comment:
It should cover `pep8` and `pyflakes`, yes. I'll remove `ament_pep8` and `ament_pyflakes` from `ament_lint_common` so that they are not added when `ament_lint_auto` is used.

I haven't added the docstrings flake8 plugin. Since you mentioned the plugins won't be installable when we build debians, perhaps that's a reason to leave `ament_pep257` as is - what do you think?